build.js

```diff
@@ -71,8 +71,8 @@ const context = await esbuild.context({
         cleanPlugin(),
         ...lint
             ? [
-                stylelintPlugin({ filter: new RegExp(cwd + '\/src\/.*\.(css?|scss?)$') }),
-                eslintPlugin({ filter: new RegExp(cwd + '\/src\/.*\.(jsx?|js?)$') })
+                stylelintPlugin({ filter: new RegExp(cwd + '/src/.*\\.(css?|scss?)$') }),
```

Comment:
Why did you add the new backlash? we need to escape the dot, and that's why we need the backlash before the dot, but I don't get why we are now escaping that.

Reply:
Heh, I suppose this is a bit subtle. Note that these are *not* regular expressions, like `/a\.b/` -- they are *strings* which get *compiled into* a RE. In a standard string, `\.` does nothing different than `.`, both are a simple period character. That's why eslint declares it as "useless":

```
> console.log(/a\.b/)
/a\.b/
> console.log('a\.b')
a.b
```

But we actually want a *RE* with `\.`, and the way to represent `\` in a string is `\\`:

```
> console.log('a\\.b')
a\.b
```
I.e. before it was wrong -- the regexp said "any character followed by css or scss" (more or less, ignoring the `?`). But we don't want that, we want "a period followed by css or scss".

Does that make sense?